automation/utilities/platform_utils.py

```diff
@@ -109,9 +110,10 @@ def get_configuration_string(manager_params, browser_params, versions):
         if item['profile_archive_dir'] is not None:
             archive_all_none = False
 
-        # Separate out long profile directory strings
+        # Separate out long configuration parameters
         profile_dirs[browser_id] = item.pop('profile_tar')
         archive_dirs[browser_id] = item.pop('profile_archive_dir')
+        js_config[browser_id] = item.pop('js_instrument_settings')
```

Comment:
The easy fix for this issue is to change this line to
```suggestion
        js_config[browser_id] = json.loads(item.pop('js_instrument_settings'))
```